src/components/EventWarning/styles.js

```diff
@@ -23,6 +23,13 @@ const styles = (theme) => ({
     background: '#E9FEFE',
     border: 'solid 1px #23DAE0',
   },
+  upcoming: {
+    background: '#F5A6231A',
+    border: 'solid 1px #F5A623',
+    fontFamily: 'Lato',
```

Comment:
can you add the font stuff to `Warning` in case we want to use `yellow` text later please? Notice how everything else just has background stuff. Just want to keep this consistent.